seleniumByTechnocredits/src/seleniumPractise/prachiJ/Assingment1/TechnoRegistration_Q6.java

```diff
@@ -0,0 +1,207 @@
+
+import java.util.ArrayList;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import util.Init;
+/*
+ * Registration> Registration > Tick button > alert
+ * Validate the alert
+ * Registration> Registration > First Name > Tick button > alert
+ * Validate the alert
+ *		and so on for every element
+ */
+public class TechnoRegistration_Q6 {
+
+	public static void main(String[] args) throws InterruptedException {
+		WebDriver driver = Init.initChromeDriver();
+		String url = "http://automationbykrishna.com/index.html";
+		driver.get(url);
+		driver.findElement(By.id("registration2")).click();
+		Thread.sleep(2000);
+		TechnoRegistration_Q6 technoRegistration_Q6 = new TechnoRegistration_Q6();
+		ArrayList<String> al = new ArrayList<String>();
+
+		WebElement fName = driver.findElement(By.xpath("//div[@id='registration']/descendant::input[@name='fName']"));
+		fName.sendKeys("Prachi Jain");
+		String fullName = fName.getAttribute("value");
+		al.add(fullName);
+
+		WebElement add = driver.findElement(By.xpath("//div[@id='registration']//following::input[position()=2][@id='address']"));
+		add.sendKeys("Atlanta 2");
+		String address = add.getAttribute("value");
+		al.add(address);
+
+		WebElement email = driver.findElement(By.xpath("//input[contains(@name,'email')]"));
+		email.sendKeys("jainprachi@gmail.com");
+		String emailId = email.getAttribute("value");
+		al.add(emailId);
+
+		WebElement city = driver.findElement(By.xpath("//input[starts-with(@placeholder,'City')]"));
+		city.sendKeys("Pune");
+		String cityTown = city.getAttribute("value");
+		al.add(cityTown);
+
+		WebElement org = driver.findElement(By.xpath("//div[@id='registration']/form/div/input[last()-3]"));
+		org.sendKeys("CG");
+		String organisation = org.getAttribute("value");
+		al.add(organisation);
+
+		WebElement genderM = driver.findElement(By.xpath("//p[text()='Enter your account details below']//preceding-sibling::div/descendant::input[1]"));
+		WebElement genderF = driver.findElement(By.xpath("//p[text()='Enter your account details below']//preceding-sibling::div/descendant::input[2]"));
+		if (genderM.isSelected() == true) {
+			genderF.click();
+		}
+
+		WebElement user = driver.findElement(By.xpath("//input[contains(@id,'username')][@class='form-control']"));
+		user.sendKeys("validateUserName");
+		String userName = user.getAttribute("value");
+		al.add(userName);
+
+		WebElement pwd = driver.findElement(By.xpath("//input[@id='passwordReg']"));
+		pwd.sendKeys("@392xyz");
+		String password = pwd.getAttribute("value");
+		al.add(password);
+
+		WebElement rePwd = driver.findElement(By.xpath("//input[contains(@id,'repasswordReg')]"));
+		rePwd.sendKeys("@392xyz");
+		String rePassword = rePwd.getAttribute("value");
+		al.add(rePassword);
+
+		WebElement terms = driver.findElement(By.xpath("//label[@class='checkbox']/input"));
```

Comment:
Hi Prachi..Getting below exception when ran your code. 
Exception in thread "main" org.openqa.selenium.WebDriverException: unknown error: Element <input type="checkbox" value="agree this condition" id="signupAgreement"> is not clickable at point (600, 704).
Could you please verify at your end as well.

Reply:
Hi Pallavi, I ran it at my end and here its working fine :)